.github/workflows/doc.yml

```
@@ -25,6 +32,26 @@ jobs:
           cargo doc --workspace --no-deps --features=permission-calculator
           cargo doc -p twilight-util --no-deps --all-features
 
+      - name: Remove examples
+        if: github.event_name == 'push' && github.ref == 'refs/heads/main'
+        run: rm -r target/doc/*_examples_*
```

we renamed this path in 1243:
```suggestion
        run: rm -r target/doc/twilight_example_*
```